lumicks/pylake/population/mixture.py

```diff
@@ -17,22 +21,23 @@ def wrapper(self, *args, **kwargs):
 
 
 class GaussianMixtureModel:
-    """A wrapper around scikit-learn's GMM.
+    """A wrapper around :class:`sklearn.mixture.GaussianMixture`.
 
-    This model accepts a 1D array as training data. The state parameters are sorted according
-    to state mean in order to facilitate comparison of models with different number of states
+    This model accepts a 1D array as training data. *The state parameters are sorted according
+    to state mean* in order to facilitate comparison of models with different number of states
     or trained on different datasets. As the current implementation is designed to specifically
-    handle 1D data, model parameters are also returned as 1D arrays (np.squeeze() is applied to the results)
-    so that users do not have to be concerned with the shape of the output results.
+    handle 1D data, model parameters are also returned as 1D arrays (:func:`numpy.squeeze()` is applied to
+    the results) so that users do not have to be concerned with the shape of the output results.
 
     Parameters
```

Comment:
Would it make sense to mention that this functionality is in alpha here? Or are we considering this API to be mature enough already? (Considering we have it marked as alpha in the tutorial).

Reply:
no, I want to keep it alpha because we haven't really worked with it much yet. good point to put it here too